src/rnr_score_hv/harvesters.py

```diff
@@ -0,0 +1,30 @@
+"""
+Copyright 2022 NOAA
+All rights reserved.
+
+Collection of methods to facilitate file/object retrieval
```

Comment:
is this where you register all available harvesters?

Reply:
Yes.  This file will grow with each additional harvester.  Are you ok with that?